src/Modules/UngradedModules/SubquoModule.jl

```diff
@@ -1619,6 +1619,130 @@ function intersect(M::SubquoModule{T}, N::SubquoModule{T}) where T
   throw(ArgumentError("Relations of M and N are not equal."))
 end
 
+########################################
+# module quotients
+########################################
+
+@doc raw"""
+     annihilator(N::SubquoModule{T}) where T
+
+Return the annihilator of `N`.
+
+!!! note
+    By definition, the annihilator of $N$ is the ideal $0:N = \{a \in R \mid aN = 0\}$. 
+    Here, `R = base_ring(N)`.
+
+# Examples
+
+```jldoctest
+julia> R, (x, y, z) = polynomial_ring(QQ, [:x, :y, :z]);
```

Comment:
Just FYI these days you could also write
```suggestion
julia> R = @polynomial_ring(QQ, [:x, :y, :z]);
```

Reply:
Thx, the next time ...